### ISSUE TITLE
Road Editing in Peru

### ISSUE THREAD

**Issue by LeeKana** [microsoft/Open-Maps#58]
### Edición de carreteras en Perú

**Descripción del Proyecto:** 

Este proyecto busca a mejorar los datos de mapas en Perú. Usaremos todos los recursos de imaginería aérea y imágenes a nivel de calle disponibles en OSM. Planeamos editar las carreteras enfocado en lo siguiente: 

  

- alineación 
- carreteras que faltan 
- carreteras sin nombre 
- etiquetas de superficie 
- clasificación de carreteras 

  

Podríamos editar algunas otras características del mapa, como cuerpos de agua, características naturales, características de uso de la tierra y edificios. 

 

Procesaremos esto de acuerdo con [OSM](https://wiki.openstreetmap.org/wiki/ES:Key:highway), las [pautas locales](https://wiki.openstreetmap.org/wiki/ES:Per%C3%BA/Convenciones) y cumpliendo con los [directrices de edición organizada](https://wiki.openstreetmap.org/wiki/ES:Directrices_de_edici%C3%B3n_organizada). 

 

 

**Pautas de Mapeo:** 

Cuando sea posible, agregaremos y actualizaremos las características de las carreterras enfocado en lo siguiente: 

- geometría 
- nombres 
- direccionalidad 
- clasificación 
- superficie (solo pavimentada o sin pavimentar) 

 

El equipo seguirá las [pautas de OSM](https://wiki.openstreetmap.org/wiki/ES:Viales) al agregar y etiquetar características del mapa. 

 

**Área de Enfoque:** 

Este será un esfuerzo a nivel nacional en Perú. 

 

 
**Comentarios en Conjuntos de Cambios:** 

El equipo agregará comentarios en conjuntos de cambios que sigan [las pautas de conjuntos de cambios de OSM](https://wiki.openstreetmap.org/wiki/ES:Buenos_comentarios_en_conjuntos_de_cambios) 

 

**Detección de Errores:** 

Usaremos [el validador de JOSM](https://wiki.openstreetmap.org/wiki/ES:JOSM/Validador) para buscar errores, siguiendo el wiki y las pautas de [LearnOSM](https://learnosm.org/es/coordination/review/). 

 

**Información adicional:** 

Para cualquier pregunta, comentario o preocupación, no dude en enviar un correo electrónico al equipo a openmaps(at)Microsoft.com.

**Comment by LeeKana** [comment]
**Task description:**  

 
In this task, we intend to improve the map data in Peru.  We will also use all available aerial and street level imagery assets available in OSM. We intend to edit the roads, looking to focus the following:  

 

- Alignment
- missing roads
- missing road names
- surface tags
- road classification 

 

We will also add or improve road relations, such as turn restrictions or destination signs. 

 

Additionally, we may edit some other map features such as water bodies, natural features, landuse features, and buildings.  

 

We will be processing this according to [OSM](https://wiki.openstreetmap.org/wiki/Key:highway), the [local guidelines](https://wiki.openstreetmap.org/wiki/ES:Per%C3%BA/Convenciones), and adhering to the [Organized editing guidelines](https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines).  

 

**Mapping Guidelines:**  

We will be adding/updating (when possible):

- road geometry
- road names
- directionality
- road classification
- restrictions
- destinations
- surface type (we will only be adding paved or unpaved)

The team will follow [OSM guidelines](https://wiki.openstreetmap.org/wiki/Highways) for adding and tagging each.  

 

**Area of Focus:**  

 

This will be a country wide effort across Peru.  


 

**Changeset Comments:** 

 

The team will be adding changeset comments that follow [OSM changeset guidelines](https://wiki.openstreetmap.org/wiki/Good_changeset_comments).  

 

**Error Detection:** 


We will use the [validator](https://wiki.openstreetmap.org/wiki/JOSM/Validator) in the JOSM tool to look for errors, following the [wiki](https://wiki.openstreetmap.org/wiki/Tasking_Manager/Validating_data) as well as the [LearnOSM guidelines](https://learnosm.org/en/coordination/review/).  

 

**Additional Info:** 

 

For any questions, comments, concerns, feel free to email the team at openmaps(at)Microsoft.com.

**Comment by erickdeoliveiraleal** [comment]
As I did for Venezuela, I created notes to help them identify missing roads: http://resultmaps.neis-one.org/osm-notes-country?c=Peru&limit=no. I used JOSM and downloaded state by state and compared with imagery to identify missing roads.

**Comment by LeeKana** [comment]
Thank you for identifying these missing roads Erick! We will work to add missing roads and resolve these map notes in Peru as well.

**Comment by ovruni** [comment]
Hello, I have noticed in Piura several street crossings with buildings, in the same way I just saw in Huaral a barrier crossing with a street. Please avoid these street crossings when adding streets in Peru.

**Comment by BrackoNe** [comment]
Hello, thanks for the post.

Please send exact samples for further investigation.
If there is a barrier and you have a local knowledge, please feel free to add a proper barrier on a node of the highway.

**Comment by ovruni** [comment]
Here are some that I just found

Sullana
way 707870466 
way 874693174 

Catacaos
way 532326293
way 863865787

Lambayeque
way 666163227
way 91014053

Puente Piedra
way 292232424
way 872866003

**Comment by BrackoNe** [comment]
> Here are some that I just found
> 
> Sullana
> way 707870466
> way 874693174
> 
> Catacaos
> way 532326293
> way 863865787
> 
> Lambayeque
> way 666163227
> way 91014053
> 
> Puente Piedra
> way 292232424
> way 872866003

Thank you so much for the quick response and for sharing such great examples!

We are aligning a network to the existing network, and even though we are trying to avoid such cases, sometimes it is not possible at all since the geometry of the buildings isn't exact, but the approximation of the footprint.

There are such bad cases like you've shared:
![image](https://user-images.githubusercontent.com/8583745/99855415-1eaa7900-2b87-11eb-8ba0-9acf4b0e32ca.png)

We will try to avoid making new logical issues in the future, but it would be great if the community can organize a mapathlon and fix such irregular buildings. We are aligning the road network to the existing one, so we are very often unable to avoid such crossings since whoever done building footprints, haven't check whether aerial imagery is aligned to the road network before they have started tracing the footprints.

Thanks once again,
Nemanja

**Comment by sanderd17** [comment]
How will you alter road alignment?
Peru has a very rough terrain, which gives issues with imagery alignment. All imagery sources I've found so far had alignment issues. You could align it on one position, but a few 100m further, it would start to deviate.

Even GPS is often imprecise next to cliffs and under trees.

I don't think bad alignment is a huge issue. If the relative alignment is correct, and features get named, this already give a whole lot of valuable information on the map. But I would refrain from aligning complete street layouts to a certain imagery source.

**Comment by BrackoNe** [comment]
I completely agree with you on this, this is why we are aligning on micro location if it is necessary.

**Comment by erickdeoliveiraleal** [comment]
Updated link: http://resultmaps.neis-one.org/osm-notes-country-custom?c=Peru&query=open
